screenshots/html/script.js

```diff
@@ -141,6 +149,25 @@ function addForm() {
   };
   label2.appendChild(input2);
   form.appendChild(label2);
+  // Add a date picker to input the minimum date of modification
+  const label3 = document.createElement('label');
```

Comment:
Definitely not a blocker, but maybe the label and input names could reference what data they're referring to?